server/sonar-process/src/main/java/org/sonar/process/cluster/info/ProcessInfoProviderRef.java

```diff
@@ -17,24 +17,25 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.server.platform.monitoring;
+package org.sonar.process.cluster.info;
 
-import java.util.Map;
 import java.util.Objects;
-import java.util.TreeMap;
+import javax.annotation.Nullable;
 
-public class JvmPropsMonitor implements Monitor {
-  @Override
-  public String name() {
-    return "JvmProperties";
+/**
+ * Trick to be able to access implementation of {@link ProcessInfoProvider}
+ * from the Hazelcast members that execute {@link ProcessInfoCall}.
+ */
+public class ProcessInfoProviderRef {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1118)